test/test.js

```diff
@@ -0,0 +1,94 @@
+var SAT = require('..')

```

Comment:
Kind of inconsistent on semicolons in this test file. Can you put them at the end of each statement?